utils/validators/validator.js

```diff
@@ -0,0 +1,15 @@
+const joi = require("joi");
+
+const contactSchema = joi.object({
+  name: joi.string().min(3),
```

Comment:
I think min should be 2 character length (because name Ed exist). And can we add max, for example 50?